src/adapters/db.ts

```diff
@@ -28,21 +54,33 @@ export function createDBComponent(components: Pick<AppComponents, 'pg' | 'logs'>
   const logger = logs.getLogger('db-component')
 
   return {
-    getFriends(userAddress, onlyActive = true) {
-      let query: SQLStatement
+    async getFriends(userAddress, { onlyActive, pagination } = { onlyActive: true }) {
+      const { limit, offset } = getPaginationOrDefaults(pagination)
```

Comment:
What about placing the default directly on this `getFriends` function's params instead at `getPaginationOrDefaults` as we do with `onlyActive:true`

Reply:
Also, could be possible that limit is set but not the offset, or vice versa?